### DISCUSSION TITLE
Add an optional CSV export on chart component

### DISCUSSION THREAD

**Discussion by pascalr0410** [sqlpage/SQLPage#60] [Ideas]
Hello,

Could it be possible to add an optional  "Download CSV" on each chart component, on the top right corner ?

It could reduce redudancy of sql and be more effiency usage of the screen space.

```sql
SELECT 'chart' as component, 'Expenses' as title, 'bar' as type, 1 as stacked, 1 as horizontal  , 
  1 as csvExport;
```

Regards,
Pascal.

**Comment by lovasoa** [comment]
Hello and thank you for opening this discussion ! 
I think it's a good idea, and since the "export csv" feature is supported in the js library that SQLPage uses for charts, this should not be too hard to do. Let's add this as a ticket to implement for next release.
